content/applications/sales/crm/pipeline/lost_opportunities.rst

```diff
@@ -2,77 +2,73 @@
-there.
+.. image:: lost_opportunities/lost-opps-lost-reason.png
+   :align: center
+   :alt: Lost reasons popup with sample reasons.
 
-Retrieve lost opportunities
-===========================
+Click :guilabel:`Submit`.
+
+Manage & create lost reasons
+============================
 
-To retrieve lost opportunities and do actions on them (send an email,
-make a feedback call, etc.), select the *Lost* filter in the search
-bar.
+To create a new lost reason, or to edit an existing one, go to :menuselection:`Configuration -->
+Lost Reasons`.
 
-.. image:: lost_opportunities/lost_opportunities02.png
-   :align: center
+Click :guilabel:`Create` and enter the new lost reason in the :guilabel:`Description` field. Click
+:guilabel:`Save`.
 
-You will then see all your lost opportunities.
+Click an existing lost reason to edit it, then click :guilabel:`Save`.
 
-If you want to refine them further, you can add a filter on the *Lost
-Reason*.
+Retrieve lost opportunities
+===========================
 
-For Example, *Too Expensive*.
+To retrieve lost opportunities, go to the pipeline and select :guilabel:`Lost` from the
+:guilabel:`Filters` menu below the search bar. To filter by a specific lost reason, select
+:menuselection:`Filters --> Add Custom Filter --> Lost Reason`.
 
-.. image:: lost_opportunities/lost_opportunities03.png
+.. image:: lost_opportunities/lost-opps-lost-filter.png
    :align: center
+   :alt: Pipeline with Lost filter in the search bar.
 
 Restore lost opportunities
 ==========================
 
-From the Kanban view with the filter(s) in place, you can select any
-opportunity you wish and work on it as usual. You can also restore it by
-clicking on *Archived*.
-
-.. image:: lost_opportunities/lost_opportunities04.png
-   :align: center
-
-You can also restore items in batch from the Kanban view when they
-belong to the same stage. Select *Restore Records* in the column
-options. You can also archive the same way.
-
-.. image:: lost_opportunities/lost_opportunities05.png
-   :align: center
-
-To select specific opportunities, you should switch to the list view.
+To restore a lost opportunity, go to the :guilabel:`Pipeline` and select :guilabel:`Lost` from the
+:guilabel:`Filters` drop-down under the search bar. Click on a kanban card to open an opportunity.
+At the top of the page, click :guilabel:`Restore`.
 
-.. image:: lost_opportunities/lost_opportunities06.png
+.. image:: lost_opportunities/lost-opps-restore.png
    :align: center
+   :alt: Lost opportunity with emphasis on the Restore button.
 
-Then you can select as many or all opportunities and select the actions
-you want to take.
+To restore multiple opportunities at once, go to the :guilabel:`Pipeline` and switch to
+:guilabel:`List View`. Select the checkbox on the left of the page for each opportunity to be
+restored. Then, click the :guilabel:`Action` gear icon at the top of the list, and select
+:guilabel:`Unarchive` from the drop-down.
```

Comment:
```suggestion
To restore multiple opportunities at once, navigate to the main :guilabel:`Pipeline` dashboard in
the *CRM* app, open the :guilabel:`Filters` drop-down menu, and select the :guilabel:`Lost` option.

Next, select the list view option, which is represented by the three-line :guilabel:`☰
list` icon in the upper-right corner. Doing so places all the leads from the :guilabel:`Pipeline`
page in a list form. With the list form chosen, select the checkbox to the left of each desired
opportunity/lead to be restored.

Once the desired opportunities/leads have been selected, click the :guilabel:`⚙️ Action` drop-down
menu at the top of the :guilabel:`Pipeline` page. From the :guilabel:`⚙️ Action` drop-down menu,
select :guilabel:`Unarchive`.

Doing so removes those selected lost opportunities from the :guilabel:`Pipeline` page because they
no longer fit into the `Lost` filter criteria. To reveal these newly-restored leads, delete the
`Lost` filter from the search bar.
```